### DISCUSSION TITLE
How to use

### DISCUSSION THREAD

**Discussion by Chisat09** [DxltaMath/DMIx#16] [Q&A] [answered]
Uh I opened Deltamath but like, what do you do from there

**Comment by afkvido** [comment]
Hi @Chisat09,

Please read our tutorial: https://github.com/DxltaMath/DMIx/blob/master/.github/meta/QUICKSTART.md

However, it's been years since I've touched the code of this extension, it's probably deprecated, and I currently have no plans to fix it in the near future, as I've been very busy.

**Reply by JackWagon885** [reply, marked as answer]
It did break
